docs/client/full-api/api/pubsub.md

```diff
@@ -173,10 +173,11 @@ collection name.
 The client will see a document if the document is currently in the published
 record set of any of its subscriptions.
 
-The `onReady` callback is called with no arguments when the server
-[marks the subscription as ready](#publish_ready). The `onError` callback is
-called with a [`Meteor.Error`](#meteor_error) if the subscription fails or is
-terminated by the server.
+The `onReady` callback is called with no arguments when the server [marks the
+subscription as ready](#publish_ready). The `onStopped` callback is called with
+a [`Meteor.Error`](#meteor_error) if the subscription fails or is terminated by
+the server. If the subscription is stopped for any other reason, `onStopped` is
```

Comment:
we can be specific --- stopped by calling `stop` on either the client or the server (right?)

are there other ways?  hmm, i actually have no idea what happens when you close a DDP connection (with the not-really-documented `connection.close()` --- does it stop the subscriptions and remove their contents from minimongo?  guessing not, though that is actually kinda weird.